fili-core/src/main/java/com/yahoo/bard/webservice/data/config/metric/makers/ConstantMaker.java

```diff
@@ -41,17 +39,26 @@ public LogicalMetric make(String metricName, List<String> dependentMetrics) {
 
     @Override
     protected LogicalMetric makeInner(String metricName, List<String> dependentMetrics) {
+        try {
+            Set<PostAggregation> postAggregations = Collections.singleton(new ConstantPostAggregation(
+                    metricName,
+                    new Double(dependentMetrics.get(0))
+            ));
 
-        Double number = new Double(dependentMetrics.get(0));
-
-        ConstantPostAggregation postAgg = new ConstantPostAggregation(metricName, number);
-
-        Set<Aggregation> aggs = Collections.emptySet();
-        Set<PostAggregation> postAggs = Collections.singleton(postAgg);
-
-        TemplateDruidQuery query = new TemplateDruidQuery(aggs, postAggs);
-
-        return new LogicalMetric(query, new NoOpResultSetMapper(), metricName);
+            return new LogicalMetric(
+                    new TemplateDruidQuery(Collections.emptySet(), postAggregations),
+                    NO_OP_MAPPER,
+                    metricName
+            );
+        } catch (NumberFormatException nfe) {
+            String message = String.format(
+                    "%s value '%s' does not parse to a number",
+                    metricName,
+                    dependentMetrics.get(0)
+            );
+            LOG.error(message);
```

Comment:
While I understand the motivation, I don't agree with it 😉 . I think there are only a small number of places the parent logger is being used.